beacon_node/execution_layer/src/lib.rs

```diff
@@ -1965,6 +1970,13 @@ fn verify_builder_bid<T: EthSpec, Payload: AbstractExecPayload<T>>(
         );
     }
 
+    let expected_withdrawals_root = payload_attributes
+        .withdrawals()
+        .ok()
+        .cloned()
+        .map(|withdrawals| Withdrawals::<T>::from(withdrawals).tree_hash_root());
```

Comment:
we can use the withdrawals root from the local payload header here and fallback to re-calculating the root when we don't have a header

Reply:
yea was talking about local payload withdrawals root (not payload attributes) root vs the relay withdrawals root. but as you pointed out offline doing it this way avoids misconstruing an issue with the local EE vs an issue with the builder